### PR TITLE
Removed Travis, minor code changes

### PR DESCRIPTION
**Pull request by nwu63**
## Purpose
I removed the old Travis files, and also made some minor changes to the yaml file. **Please review those changes since I'm not as familiar with how Azure works**

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

## Testing
We can't really test this..

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

**Review by sseraj — COMMENTED** [review]
I would remove the line about Travis in the top level README. The Docker changes look good.

File: azure/azure_build.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ jobs:
     value: ${{ variables.MDOLAB_USER }}/repos/${{ parameters.REPO_NAME }}
   - name: DOCKER_MOUNT_DIR
     value: ${{ variables.MDOLAB_USER }}/azure/${{ parameters.REPO_NAME }}
+  - name: BASHRC
+    value: \$HOME/.bashrc_mdolab
```
[the remark this answers is not in the document]
Comment (nwu63) [reply azure/azure_build.yaml:45]: Oh I see, good point

[the remark this answers is not in the document]
Comment (nwu63) [reply azure/azure_build.yaml]: I renamed `MDOLAB_USER` to `MDOLAB_HOMEDIR`

[the remark this answers is not in the document]
Comment (sseraj) [reply azure/azure_build.yaml]: That's a better name

File: azure/azure_build.yaml [hunk 2]
```diff
@@ -35,14 +35,14 @@ jobs:
...
     value: /home/mdolabuser/
   - name: DOCKER_WORKING_DIR
-    value: ${{ variables.MDOLAB_USER }}/repos/${{ parameters.REPO_NAME }}
+    value: ${{ variables.MDOLAB_HOMEDIR }}/repos/${{ parameters.REPO_NAME }}
   - name: DOCKER_MOUNT_DIR
-    value: ${{ variables.MDOLAB_USER }}/azure/${{ parameters.REPO_NAME }}
+    value: ${{ variables.MDOLAB_HOMEDIR }}/azure/${{ parameters.REPO_NAME }}
   - name: BASHRC
-    value: \$HOME/.bashrc_mdolab
+    value: ${{ variables.MDOLAB_HOMEDIR }}.bashrc_mdolab
```
Comment (sseraj) [inline azure/azure_build.yaml:45]: Missing the slash

Comment (nwu63) [reply azure/azure_build.yaml:45]: Ugh dammit. Thank you for being thorough.

Comment (sseraj) [reply azure/azure_build.yaml:45]: Oh I see, you the variable has a slash

Comment (nwu63) [reply azure/azure_build.yaml:45]: Yeah I should probably remove that slash right?

Comment (sseraj) [reply azure/azure_build.yaml:45]: I've actually been wanting to fix that. There are duplicate slashes in most cases.

Comment (sseraj) [reply azure/azure_build.yaml:45]: I'm a few seconds behind. Yes, I agree.

**Review by sseraj — APPROVED** [review]

**Review by bernardopacini — APPROVED** [review]

**Pull request merged**